solidity/test/LPRewardsTest.js

```diff
@@ -0,0 +1,119 @@
+const {contract, web3} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("./helpers/snapshot")
+const {accounts} = require("@openzeppelin/test-environment")
+
+const LPRewards = contract.fromArtifact("LPRewards")
+const KeepToken = contract.fromArtifact("KeepToken")
+const LPToken = contract.fromArtifact("TestToken")
```

Comment:
```suggestion
const LPWrappedToken = contract.fromArtifact("TestToken")
```

Reply:
Yeah, second thoughts I mentioned in the other comment. Let's align the name with the one we decide to use in the contract.

Reply:
Done, aligned with the one in contract